Taskfile.yml

```diff
@@ -3,70 +3,63 @@
 
 version: "3"
 
-env:
-  DOCKER_IMAGE: cogito
-
 vars:
   DATE: '{{ now | date "2006-01-02" }}'
   COMMIT:
     sh: git log -n 1 --format=%h
   REPO: github.com/Pix4D/cogito
-  TAG:
+  DOCKER_TAG:
     sh: ci/git-ref-to-docker-tag.sh
-  BUILD_INFO: "Tag: {{.TAG}}, commit: {{.COMMIT}}, build date: {{.DATE}}"
+  DOCKER_ORG: '{{default "pix4d" .DOCKER_ORG}}'
+  DOCKER_IMAGE: cogito
+  DOCKER_BASE_NAME: "{{.DOCKER_ORG}}/{{.DOCKER_IMAGE}}"
+  DOCKER_FULL_NAME: "{{.DOCKER_BASE_NAME}}:{{.DOCKER_TAG}}"
+  BUILD_INFO: "Tag: {{.DOCKER_TAG}}, commit: {{.COMMIT}}, build date: {{.DATE}}"
   LDFLAGS: -w -X '{{.REPO}}/resource.buildinfo={{.BUILD_INFO}}'
   GOTESTSUM:
     sh: "echo $(which gotestsum 2> /dev/null)"
   TESTRUNNER: "{{if .GOTESTSUM}}{{base .GOTESTSUM}} --{{else}}go test{{end}}"
+  #
+  GOLANGCI_VERSION: v1.42.1
+  GOTESTSUM_VERSION: v1.7.0
 
 tasks:
-  default:
-    deps: [test]
-    desc: Sensible defaults if you invoke `task` without arguments.
 
-  ci-init:
-    desc: Useful only when running under CI.
+  install:deps:
+    desc: Install tool dependencies.
     cmds:
-      - cmd: go get gotest.tools/gotestsum
-      - cmd: echo
-        silent: true
-      - cmd: go mod download
+      - go install github.com/golangci/golangci-lint/cmd/golangci-lint@{{.GOLANGCI_VERSION}}
+      - go install gotest.tools/gotestsum@{{.GOTESTSUM_VERSION}}
 
-  test:
+  lint:
+    desc: Lint the code.
+    cmds:
+      # The flag "--enable" actually _adds_ to the default linters
+      - golangci-lint run --enable gofmt,gocritic ./...
+
+  test:unit:
     desc: Run the unit tests.
     cmds:
-      - "{{.TESTRUNNER}} -count=1 -coverprofile=coverage.out ./..."
-      - go tool cover -html=coverage.out -o coverage.html
+      - "{{.TESTRUNNER}} -count=1 -short -coverprofile=coverage.out ./..."
 
   test:env:
     desc: "Run what is passed on the command-line with a shell environment containing the secrets needed for the integration tests. Example: task test:env -- go test -count=1 -run '^TestFooIntegration' ./pkg/update"
     cmds:
       - '{{ .CLI_ARGS }}'
     env: &test-env
-      COGITO_TEST_COMMIT_SHA:
-        sh: gopass show cogito/test_commit_sha
+      COGITO_TEST_COMMIT_SHA: '{{default "32e4b4f91bb8de500f6a7aa2011f93c3f322381c" .COGITO_TEST_COMMIT_SHA}}'
```

Comment:
Removed from secret store what is not actually a secret.
This Task syntax allows a contributor to use his/her own integration test setup by setting the $COGITO_TEST_COMMIT_SHA before calling Task.